src/pages/sangcheol/Main/Main.js

```diff
@@ -1,8 +1,83 @@
 import React from 'react';
+import Header from './Components/Header';
+import Feed from './Components/Feed';
+import MainRight from './Components/MainRight';
+import './Main.scss';
+import './Main-section.scss';
+import '../../../styles/reset.scss';
+import '../../../styles/common.scss';
 
 class Main extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      inputKeyword: '',
+      commentBox: [],
+    };
+  }
+
+  handleInput = e => {
+    console.log(e.target.name);
+    this.setState({
+      inputKeyword: e.target.value,
+    });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    if (this.state.inputKeyword.length === 0) return;
+
+    this.setState({
+      commentBox: this.state.commentBox.concat(this.state.inputKeyword),
```

Comment:
```suggestion
      commentBox: [...commentBox, inputKeyword]
```
- 구조분해 할당 잘 해주시고
- 스프레드 연산자 사용하셔도 같습니다